src/main/webapp/index.html

```diff
@@ -1,15 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This file is deleted now, but in https://github.com/googleinterns/step217-2020/pull/1/files we modified it to include the login logic. Make sure to rebase this PR to include the login functionality

Reply:
Done in https://github.com/googleinterns/step217-2020/pull/5